applications/luci-app-bcp38/htdocs/luci-static/resources/view/bcp38/form.js

```diff
@@ -0,0 +1,65 @@
+/*
+LuCI - Lua Configuration Interface
+
+Copyright 2014 Toke Høiland-Jørgensen <toke@toke.dk>, 2023 S.Brusch <ne20002@gmx.ch>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+$Id$
+*/
```

Comment:
Can you reduce the size of the header? Take a look at other javascript files in this repository.
Search for `Copyright`. Additional this not a `lua` file